cql3/query_processor.cc

```diff
@@ -42,6 +42,15 @@ const sstring query_processor::CQL_VERSION = "3.3.1";
 
 const std::chrono::minutes prepared_statements_cache::entry_expiry = std::chrono::minutes(60);
 
+struct query_processor::remote {
```

Comment:
The word "remote" seems not very nice here, as queries are either local or distributed, not remote. Maybe call it "distributed"?

Reply:
It's consistent with `storage_proxy::remote`

Reply:
The seastar::distributed is sort of implicitly deprecated :\